sprime/sprime.py

```diff
@@ -0,0 +1,38 @@
+"""
+ID: whuan2001
+LANG: PYTHON2
+TASK: sprime
+"""
+import math
+def solution():
+    fin = open("sprime.in", "r")
+    fout = open("sprime.out", "w")
+    N = int(fin.readline().strip())
+    primes = []
+    for res in creat_primes(N, N, primes):
+        print >>fout, res
+def creat_primes(N, num, primes):
```

Comment:
General question: you just need to keep the last recursion instead of keep everything in a 2d array.